src/d/actor/d_a_wbird.cpp

```diff
@@ -5,70 +5,208 @@
+BOOL daWbird_c::CreateInit() {
+    tevStr.mRoomNo = current.roomNo;
+    mAction = 3;
+    fopAcM_orderChangeEvent(this, "TACT_WINDOW", 0, -1);
+    field_0x29E = 0;
+    calcMtx();
+    return TRUE;
 }
 
 /* 00000388-000003E0       .text create__9daWbird_cFv */
 cPhs_State daWbird_c::create() {
-    /* Nonmatching */
+    fopAcM_SetupActor(this, daWbird_c);
+    CreateInit();
+    return cPhs_COMPLEATE_e;
 }
 
 /* 000003E0-000003E4       .text actionWait__9daWbird_cFv */
 void daWbird_c::actionWait() {
-    /* Nonmatching */
 }
 
 /* 000003E4-00000474       .text actionEnd__9daWbird_cFv */
 void daWbird_c::actionEnd() {
-    /* Nonmatching */
+    if (dComIfGp_evmng_endCheck(mEventIdx)){
+        dComIfGp_event_reset();
+        fopAcM_delete(this);
+    } else {
+        dComIfGp_evmng_cutEnd(dComIfGp_evmng_getMyStaffId("WINDMAN"));
+    }
 }
 
 /* 00000474-000005DC       .text actionMove__9daWbird_cFv */
 void daWbird_c::actionMove() {
-    /* Nonmatching */
+    daPy_py_c* player = (daPy_py_c*)daPy_getPlayerActorClass();
+    if(field_0x29D){
+        field_0x29D = false;
+        mDoAud_seStart(JA_SE_TAKT_WIND_DEMO);
+    }
+    if (field_0x29E > 0) {
+        field_0x29E--;
+        if (field_0x29E > 60) {
+            fopAcM_posMove(this, NULL);
+            speed.y = speed.y + field_0x2A0;
+        }
+        if (field_0x29E < 15) {
+            dKyw_custom_windpower(0.0f);
+        }
+        if (field_0x29E == 70) {
+            dKyw_tact_wind_set_go();
+        }
+    } else {
+        dComIfGp_evmng_cutEnd(dComIfGp_evmng_getMyStaffId("WINDMAN"));
+        player->changeDemoMoveAngle(mAngle);
+        mAction = 1;
+        mDoAud_seStart(JA_SE_TAKT_WIND_END);
+    }
 }
 
 /* 000005DC-00000850       .text actionSelect__9daWbird_cFv */
 void daWbird_c::actionSelect() {
-    /* Nonmatching */
+    short sVar2 = field_0x29E;
+    if (sVar2 == 10) {
+        dComIfGp_setOperateWindOn();
+        mDoAud_seStart(JA_SE_TAKT_WIND_DISP);
+        field_0x29E++;
+    } else if(sVar2 > 10){
+        mDoAud_seStart(JA_SE_SYS_WTAKT_WIND_AMB);
+        switch (dComIfGp_getOperateWind()) {
+            case 1: {
+                mDoAud_seStart(JA_SE_TAKT_WIND_DECIDE);
+                setStartPos();
+                mAction = 2;
+                daPy_py_c* player = daPy_getPlayerActorClass();
+                sVar2 = current.angle.y + 0x7fff;
+                mAngle = player->shape_angle.y;
+                player->setPlayerPosAndAngle(&player->current.pos, sVar2);
+                if(dComIfGp_checkPlayerStatus0(0, 0x00010000)){
+                    mEventIdx = dComIfGp_evmng_getEventIdx("TACT_WINDOW2_SHIP");
+                } else{
+                    mEventIdx = dComIfGp_evmng_getEventIdx("TACT_WINDOW2");
+                }
+                fopAcM_orderChangeEventId(this, mEventIdx, 0, 0xFFFF);
+                player = daPy_getPlayerActorClass();
+                player->cancelOriginalDemo();
+                dKyw_custom_windpower(windpower);
```

Comment:
Was pulling `windpower` out as a constant instead of writing
```suggestion
                dKyw_custom_windpower(1.0f);
```
an intentional choice?

If you do want it as a constant, I think it should have its type changed, since the parameter to dKyw_custom_windpower is not an int:
```diff
-const int windpower = 1;
+const f32 windpower = 1.0f;
```